operator-framework-core/src/test/java/io/javaoperatorsdk/operator/processing/EventDispatcherTest.java

```diff
@@ -121,6 +139,8 @@ void callsDeleteIfObjectHasFinalizerAndMarkedForDelete() {
 
   /** Note that there could be more finalizers. Out of our control. */
   @Test
+  @Disabled(
```

Comment:
We should not disable these tests. Either should be fixed, or if does not make sense removed.